src/gui/uithememanager.cpp

```diff
@@ -140,3 +154,81 @@ QString UIThemeManager::getIconPath(const QString &iconId) const
 #endif
     return IconProvider::instance()->getIconPath(iconId);
 }
+
+void UIThemeManager::loadColorsFromJSONConfig()
+{
+    const QString configFileName(":uitheme/config.json");
```

Comment:
Shouldn't it be constant CONFIG_FILE_NAME in anonymous namespace avove?

Reply:
`CONFIG_FILE_NAME ` outside this function doesn't make much sense (considering the name) (convey its meaning) but I will change it

Reply:
>CONFIG_FILE_NAME outside this function doesn't make much sense (considering the name) (convey its meaning)

Why is name bad?
Anyway there are several names that depend on internal theme folder name. IMO, it's better to declare such names using some predefined constant for common part.